go/test/endtoend/vreplication/r

```diff
@@ -0,0 +1,50 @@
+cleanup() {
+    rm -rf /Users/rohit/vtdataroot/*
```

Comment:
This looks wrong. Do we want to use an env variable `$VTDATAROOT` instead.

Reply:
Mistakenly added my local test scripts. Removed.